docs/shared/migration/adding-to-existing-project.md

```diff
@@ -322,21 +302,30 @@ To connect to Nx Cloud:
 - Commit and push your changes
```

Comment:
The instructions should be more like for React/Angular monorepo tutorials. Make sure they create new GitHub repo, push their local repo, and then run `nx connect`. We don't need them to go to cloud.nx.app first with the new onboarding flow.